DateTime.js

```diff
@@ -275,7 +279,7 @@ var Datetime = React.createClass({
 			children = [ DOM.input( assign({
 				key: 'i',
 				type:'text',
-				className:'form-control',
+				className: this.props.className,
```

Comment:
Hi @anateus 

Sorry about my late reply, I've been very busy :)

No problem about the polyfill or not using the dataset, but there is already a way of setting the className for the input using the prop `inputProps`. 
```js
<Datetime inputProps={ {className: 'myAwesomeClass' } } />
```
I would reserve the `className` prop for the component wrapper instead. If you modify this I will merge the PR.

Thanks for your collaboration, it is really worthy :)